tests/e2e-ibc/main_test.go

```diff
@@ -0,0 +1,202 @@
+package main_test
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	"cosmossdk.io/math"
+	transfertypes "github.com/cosmos/ibc-go/v8/modules/apps/transfer/types"
+
+	"github.com/strangelove-ventures/interchaintest/v8"
+	"github.com/strangelove-ventures/interchaintest/v8/chain/cosmos"
+	"github.com/strangelove-ventures/interchaintest/v8/ibc"
+	"github.com/strangelove-ventures/interchaintest/v8/testreporter"
+
+	"github.com/stretchr/testify/require"
+	"go.uber.org/zap/zaptest"
+)
+
+func TestInterchainIBC(t *testing.T) {
+	ctx := context.Background()
+
+	// setup chains
+	cf := interchaintest.NewBuiltinChainFactory(zaptest.NewLogger(t), []*interchaintest.ChainSpec{
+		{
+			Name: "kava",
+			ChainConfig: ibc.ChainConfig{
+				Type:           "cosmos",
+				ChainID:        "kava_8888-1",
+				Images:         []ibc.DockerImage{{Repository: "ghcr.io/strangelove-ventures/heighliner/kava", Version: "v0.26.0", UidGid: "1025:1025"}},
```

Comment:
here is the hardcoded v0.26.0 kava image we'll want to build from current branch code for CI